python/tests/test_genotype_matching.py

```diff
@@ -1345,12 +1345,17 @@ def verify(self, ts):
             self.assertAllClose(ll_tree, ll_mirror_tree_dict)
 
             # Ensure that the decoded matrices are the same
+            flipped_G_check = np.flip(G_check, axis=0)
+            flipped_s = np.flip(s, axis=1)
+            num_alleles = ls.core.get_num_alleles(flipped_G_check, flipped_s)
```

Comment:
This seems a bit backwards, requiring the client code to call into a private method in lshmm. How about we make ``num_alleles`` an optional argument in ``lshmm``, which if not provided calls ``get_num_alleles`` on the reference panel and query string?

Reply:
Also the reason this is failing is because you've not imported ``lshmm.core``

Reply:
I'm planning on getting rid of that `num_alleles` argument altogether in the next release lshmm. It is done this way right now because of how input data are being handled in the diploid case.

Reply:
Ok, want to fixup here so by importing lshmm.core?

Reply:
I think I'll get the next release of lshmm ASAP today.

Reply:
Hopefully, `0.0.7` will take care of it. https://github.com/astheeggeggs/lshmm/releases/tag/v0.0.7

Reply:
Great - I think you probably need to update the version pins in the various ``requirements`` files. Do a search for lshmm in there

Reply:
Updated tests and pins.